evm_arithmetization/src/cpu/kernel/tests/transient_storage.rs

```diff
@@ -110,6 +110,7 @@ fn test_tstore_tload() -> Result<()> {
     interpreter.generation_state.memory.set(addr_addr, 3.into());
 
     interpreter.run()?;
+    println!("MEjor aca");
```

Comment:
:/

Reply:
ouch

Reply:
I've removed them from the `feat/cancun` branch.

Reply:
thanks!